src/app/content/components/utils/scrollToContent.ts

```diff
@@ -0,0 +1,21 @@
+import scrollTo from 'scroll-to-element';
+import { mobileBreak } from '../../../theme';
+import {
+  bookBannerDesktopMiniHeight,
+  bookBannerMobileMiniHeight,
+  toolbarDesktopHeight,
+  toolbarMobileHeight
+} from '../constants';
+
+const pxPerRem = 10;
+// height of navbar in px
+const desktopOffset = -pxPerRem * (bookBannerDesktopMiniHeight + toolbarDesktopHeight);
+const mobileOffset = -pxPerRem * (bookBannerMobileMiniHeight + toolbarMobileHeight);
+
+export default function scrollToContent(elem: Element | string) {
+    // innerWidth works best on Chrome
+    // check other browsers at https://ryanve.com/lab/dimensions/
+    // default to desktopOffset if window is undefined (at least the target is always visible)
+    const offset = window && window.innerWidth <= pxPerRem * mobileBreak ? mobileOffset : desktopOffset;
```

Comment:
this is what `window.matchMedia(theme.breakpoints.mobileQuery)` is for

but as i wrote above, i think it would be better to `window.getComputedStyle(document.documentElement).scrollPaddingTop`

Reply:
I will change it to use `getComputedStyle`. This should also fix the missing coverage for this statement.